server/ai_live_video.go

```diff
@@ -152,6 +152,9 @@ func startControlPublish(control *url.URL, params aiRequestParams) {
 		ControlPub:  controlPub,
```

Comment:
I think we should probably move this initialisation of the LivePipeline somewhere more central now that we're using it to check stream existence as well as just storing these control related fields. Also do we need a mutex around it like we do in `cleanupLive()` @j0sh ?

Reply:
Up to you... I don't see a big issue right now. We might outgrow it at some point but if you want to be preemptive about refactoring, go ahead. Just want to avoid spreading initialization logic around too much, while it is kinda self contained right now.

>  Also do we need a mutex around it like we do in cleanupLive()

Yep we need the mutex but I believe we already hold it at this point. Only slight issue is that we also hold the mutex while calling `stats.Record` but that should be OK for the most part. Recording prom metrics is supposed to be fairly quick but its still kind of non-trivial IIRC

Reply:
Oh yeah missed the mutex 👍